scss/general.scss

```
@@ -221,6 +222,9 @@ article {
 
   scroll-margin-top: var.$menu-bar-height;
 
+  content-visibility: auto;
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)